cypress/e2e/pages/testsForAllPages.js

```diff
@@ -28,6 +29,32 @@ export const testsThatAlwaysRunForAllPages = ({
       }
     });
 
+    it('should have href values correctly formed', () => {
+      const masterbrandOverrides = [
+        'oromo',
+        'bangla',
+        'indonesian',
+        'ukrainian',
+        'cantonese',
+        'ws',
+      ];
+
+      cy.get('a[href^="https://www.bbc.com"]').each($tag => {
+        if ($tag.closest('footer').length > 0) {
+          return;
+        }
+        const href = $tag.attr('href');
+        expect(href).to.exist;
+        expect(href).to.not.be.empty;
+
+        const containsValidService =
+          href.includes(service) ||
```

Comment:
Should we also validate that it's one of our WS services? Some pages reference other WS services which are related